### PR TITLE
[RW-4856][risk=no] Disable beta access banner based on server-side flag

### PR DESCRIPTION
**Pull request by gjuggler**
Pretty straightforward change to hide the beta banner in environments where we're not using the beta access module.

Before:

Screen Shot 2020-04-30 at 11 27 56 PM

After:

Screen Shot 2020-04-30 at 11 23 33 PM

---
**PR checklist**

- [X] This PR meets the Acceptance Criteria in the JIRA story
- [X] The JIRA story has been moved to Dev Review
- [X] This PR includes appropriate unit tests
- [X] I have run and tested this change locally

### CODE REVIEW

File: ui/src/app/pages/admin/admin-user-bypass.tsx [hunk 1]
```diff
@@ -68,12 +68,16 @@ export class AdminUserBypass extends React.Component<
...
+      enableComplianceTraining,
+      enableEraCommons,
+      enableDataUseAgreement} = serverConfigStore.getValue();
     return <PopupTrigger
         side='bottom'
         onClose={() => {this.cancel(); this.setState({open: false}); }}
         onOpen={() => this.setState({open: true})}
         content={<FlexColumn style={{padding: '1rem'}}>
-          <FlexRow style={{justifyContent: 'space-between'}}>
+          {enableBetaAccess && <FlexRow style={{justifyContent: 'space-between'}}>
```
Comment (calbach) [inline ui/src/app/pages/admin/admin-user-bypass.tsx:80]: nit: probably group this and the next element into a `<React.Fragment>` so you just have a single conditional check. Example: https://github.com/all-of-us/workbench/blob/676d65fd44f9ff9fc09f599fdb204b84330630a8/ui/src/app/pages/profile/data-user-code-of-conduct.tsx#L130-L152

File: ui/src/app/pages/homepage/homepage.tsx [hunk 2]
```diff
@@ -186,7 +186,7 @@ export const Homepage = withUserProfile()(class extends React.Component<Props, S
         reload();
       }, 10000);
     } else {
-      if (!profile.betaAccessRequestTime) {
+      if (serverConfigStore.getValue().enableBetaAccess && !profile.betaAccessRequestTime) {
```
Comment (calbach) [inline ui/src/app/pages/homepage/homepage.tsx:189]: Good catch

Comment (gjuggler) [reply ui/src/app/pages/homepage/homepage.tsx:189]: This was Joel's catch actually – this PR builds off of his https://github.com/all-of-us/workbench/commit/66a85ec70ecdc7984ed82f89b1aff8966de0d9c2

**Comment by gjuggler** [comment]
Oops – need to close this one, I ran into some rebase issues (user error). Will reopen on a new branch.

**Pull request closed without merging**